igvc_navigation/launch/mapper.launch

```diff
@@ -7,8 +7,12 @@
             <param name="occupancy_grid_length" type="int" value="50" />
             <param name="occupancy_grid_width" type="int" value="50" />
             <param name="occupancy_grid_resolution" type="double" value="0.2" />
+            <param name="occupancy_grid_threshold" type="int" value="178" />
+            <param name="decay_period" type="double" value="0.5" />
```

Comment:
This parameter should be configurable to allow for no decay at all. Consider a way to declare `decay_period` such that the map doesn't decay at all (perhaps a negative value).